x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/job/messages/Messages.java

```diff
@@ -148,7 +148,7 @@
             "Invalid update value for analysis_limits: model_memory_limit cannot be decreased below current usage; " +
                     "current usage [{0}], update had [{1}]";
     public static final String JOB_CONFIG_DUPLICATE_DETECTORS_DISALLOWED =
-            "Specifying duplicate detectors is disallowed, {0}";
+            "Duplicate detectors are not allowed, see ''{0}''";
```

Comment:
Hm, perhaps we can do `"Duplicate detectors are not allowed: [{0}]"`. We prefer putting expressions in square brackets though in `Messages` there are still some messages that don't do that as heritage from the prelert days.

Reply:
Done.